src/commands/openReferenceBeside.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { commands, window, Selection, ViewColumn } from 'vscode';
+
+import openReferenceBeside from './openReferenceBeside';
+import {
+  closeEditorsAndCleanWorkspace,
+  createFile,
+  openTextDocument,
+  rndName,
+  toPlainObject,
+} from '../test/testUtils';
+
+describe('openReferenceBeside command', () => {
+  beforeEach(closeEditorsAndCleanWorkspace);
+  afterEach(closeEditorsAndCleanWorkspace);
+
+  it('should call open command-line tool when editor selection is within the reference', async () => {
```

Comment:
```suggestion
  it('should execute vscode.open command when editor selection is within the reference', async () => {
```

as far as I understand it does not call [open](https://github.com/svsool/vscode-memo/blob/master/src/commands/openReferenceInDefaultApp.ts#L19) command-line tool, open command line tool is used only as part of `openReferenceInDefaultApp.ts`